docs/rollup.md

```diff
@@ -1,38 +1,37 @@
-| <a id="rollup_bundle-is_windows"></a>is_windows |  Internal use only.   | Boolean | optional | <code>False</code> |
-| <a id="rollup_bundle-output_dir"></a>output_dir |  Whether to produce a directory output. We will use the [<code>--output.dir</code> option](https://github.com/rollup/rollup/blob/master/docs/999-big-list-of-options.md#outputdir) in rollup rather than <code>--output.file</code>. If the program produces multiple chunks, you must specify this attribute. Otherwise, the outputs are assumed to be a single file.   | Boolean | optional | <code>False</code> |
-| <a id="rollup_bundle-rollup"></a>rollup |  Target that executes the rollup binary   | <a href="https://bazel.build/concepts/labels">Label</a> | optional | <code>@rollup</code> |
-| <a id="rollup_bundle-silent"></a>silent |  Whether to execute the rollup binary with the --silent flag, defaults to False. Using --silent can cause rollup to [ignore errors/warnings](https://github.com/rollup/rollup/blob/master/docs/999-big-list-of-options.md#onwarn)  which are only surfaced via logging.  Since bazel expects printing nothing on success, setting silent to True is a more Bazel-idiomatic experience, however could cause rollup to drop important warnings.   | Boolean | optional | <code>False</code> |
-| <a id="rollup_bundle-silent_on_success"></a>silent_on_success |  Even stronger than --silent, defaults to False. Since the build still emits some texted, even when passed --silent, this uses the same flag as npm_package_bin to supress all output on sucess.   | Boolean | optional | <code>False</code> |
-| <a id="rollup_bundle-sourcemap"></a>sourcemap |  Whether to produce sourcemaps. Passed to the [<code>--sourcemap</code> option](https://github.com/rollup/rollup/blob/master/docs/999-big-list-of-options.md#outputsourcemap") in Rollup   | String | optional | <code>"inline"</code> |
-| <a id="rollup_bundle-srcs"></a>srcs |  Non-entry point JavaScript source files from the workspace. You must not repeat file(s) passed to entry_point/entry_points.   | <a href="https://bazel.build/concepts/labels">List of labels</a> | optional | <code>[]</code> |
+| <a id="rollup-name"></a>name |  A unique name for this target.   | <a href="https://bazel.build/concepts/labels#target-names">Name</a> | required |  |
+| <a id="rollup-args"></a>args |  -   | List of strings | optional | <code>[]</code> |
```

Comment:
Is this expected? All the docs are gone...

Reply:
Huh. Nice catch. Something up with the formatting.

Reply:
Fixed